organization/summit.md

```diff
@@ -0,0 +1,23 @@
+# Mentor Summit
+
+Every year Google promote a Summit
+and each [**mentoring**] organization,
+**in this case only NumFOCUS**,
+may send two (2) members to the Mentor Summit.
+
+NumFOCUS will nominate as Summit ambassador
+the lead administrator of the current year
+**and** the lead administrator of following year.
```

Comment:
yes I would say that we have two admins now and if the two of them are interested in going they should have first pick.